core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,32 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.generateDummyExplorations = function() {
+          // Generate dummy explorations with random title.
+          AdminTaskManagerService.startTask();
+          var DummyExpCount = $scope.DummyExplorationCount,
```

Comment:
I've mentioned this before, but the capitalization is incorrect. It should be $scope.dummyExplorationCount, $scope.dummyExplorationPublishCount.

Or, better: $scope.numDummyExpsToGenerate, $scope.numDummyExpsToPublish as mentioned above.

You don't need these to be var, btw. By "initialize" I meant that, in the controller, outside this function, you write:

    $scope.numDummyExpsToPublish = 0;
    $scope.numDummyExpsToGenerate = 0;